src/styles/block/main.scss

```diff
@@ -0,0 +1,27 @@
+.catalog {
+  --column-count: 1;
+
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: repeat(var(--column-count), $product-card-width);
```

Comment:
The variable `$product-card-width` is used here. Ensure that this variable is defined in your SCSS files. If it's not defined, it will cause a compilation error.